fluster/decoders/gstreamer.py

```diff
@@ -87,6 +87,15 @@ class GStreamer010(GStreamer):
     provider = 'GStreamer'
 
 
+@register_decoder
+class GStreamerLibavH264(GStreamer10):
+    '''GStreamer H.264 Libav decoder implementation for GStreamer 1.0'''
+    codec = Codec.H264
+    decoder_bin = ' h264parse ! avdec_h264 '
+    api = 'Libav'
+    hw_acceleration = True
```

Comment:
Shouldn't hw_acceleration be False for Libav?

Reply:
right